ci/taos/plugins-staging/pr-audit-nnstreamer-ubuntu-apptest.sh

```diff
@@ -91,6 +91,14 @@ function pr-audit-nnstreamer-ubuntu-apptest-run-queue() {
     echo -e "[DEBUG] LD_LIBRARY_PATH is '$LD_LIBRARY_PATH'"
     echo -e "[DEBUG] GST_PLUGIN_PATH is '$GST_PLUGIN_PATH'"
 
+    # nnstreamer env variables and include paths
+    export NNSTREAMER_CONF=$NNST_ROOT
+    export NNSTREAMER_FILTERS=$NNST_ROOT/lib/nnstreamer/filters
+    export NNSTREAMER_DECODERS=$NNST_ROOT/lib/nnstreamer/decoders
+    export NNSTREAMER_CUSTOMFILTERS=$NNST_ROOT/lib/nnstreamer/customfilters
+    export C_INCLUDE_PATH=$C_INCLUDE_PATH:$NNST_ROOT/include
+    export CPLUS_INCLUDE_PATH=$CPLUS_INCLUDE_PATH:$NNST_ROOT/include
```

Comment:
How about we use the below naming style. 
**s/CPLUS/CPP/g**

Reply:
it is predefined name to search inc files (https://gcc.gnu.org/onlinedocs/cpp/Environment-Variables.html)

Reply:
Got it. Thanks.

Reply:
https://github.com/nnsuite/nnstreamer-example/issues/14